src/test/groovy/com/lesfurets/jenkins/TestRunClosure.groovy

```diff
@@ -0,0 +1,47 @@
+package com.lesfurets.jenkins
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.junit.Test
+
+import static com.lesfurets.jenkins.unit.MethodCall.callArgsToString
+import static org.junit.Assert.assertTrue
+
+class TestRunClosure extends BasePipelineTest {
+
+    @Override
+    @Before
+    void setUp() throws Exception {
+        super.setUp()
+    }
+
+    @Test
+    void should_execute_without_errors() throws Exception {
+        runScript({ script ->
```

Comment:
I like the idea. Can I just use something like this?
```
runScript({
  echo "Hello World!"
})
```

Reply:
It is exactly what I was trying to do in the first place, but I could not find a way to plug the framework on the closure (see [these 3 lines](https://github.com/jenkinsci/JenkinsPipelineUnit/pull/168/files#diff-517acc3404332fdd79708acaa681880eR191-R193)). It fails with this error:
`No signature of method: com.lesfurets.jenkins.TestRunClosure.echo()`